library/aws/alarms.ts

```diff
@@ -236,7 +236,7 @@ export function createEc2HighMemAlarm(
     dimensions: {
       InstanceId: ec2.id.value,
     },
-    alarm_description: 'Sustained high memory for application server',
```

Comment:
@timbod7 @paul-thompson-helix I added only three lines above for better alarm name but this PR contains whole set which plp made over time.

You are ok with merging in one go?

Presumably they were already reviewed/agreed

Reply:
The commits:

* Add more r5 instances
* better description for alarm

both look fine. I have no idea about the other one.

Reply:
> Presumably they were already reviewed/agreed

They are not on master, so why do you presume they were reviewed?